projects/demo/src/modules/components/radio-block/examples/2/index.html

```diff
@@ -74,7 +74,43 @@
             item="pineapple"
             size="l"
         >
-            Pinapples
+            Pineapples
         </tui-radio-block>
     </div>
 </form>
+<h3
+    i18n
+    class="title"
+>
+    Group with multiple values
+</h3>
+
+<p>
+    You can make group behavior with multiple checked values by
+    <code>CheckboxBlock</code>
+    component
+</p>
+
+<form
+    class="group"
+    [formGroup]="groupForm"
+>
+    <div
+        tuiGroup
+        class="group"
+        [collapsed]="true"
+    >
+        <tui-checkbox-block
```

Comment:
Maybe it is better to put this example here?

https://taiga-ui.dev/components/checkbox-block#groups

Reply:
You've put example of `CheckboxBlock` in `RadioBlock`-page

Reply:
<img width="506" alt="image" src="https://user-images.githubusercontent.com/12021443/213186139-e848fb79-382a-496a-a922-7a464922969c.png">